util.js

```diff
@@ -703,13 +703,21 @@ function callbackify(original) {
     // In true node style we process the callback on `nextTick` with all the
     // implications (stack, `uncaughtException`, `async_hooks`)
     original.apply(this, args)
-      .then(function(ret) { process.nextTick(cb.bind(null, null, ret)) },
-            function(rej) { process.nextTick(callbackifyOnRejected.bind(null, rej, cb)) });
+      .then(function (ret) { process.nextTick(cb.bind(null, null, ret)) },
+        function (rej) { process.nextTick(callbackifyOnRejected.bind(null, rej, cb)) });
   }
 
   Object.setPrototypeOf(callbackified, Object.getPrototypeOf(original));
-  Object.defineProperties(callbackified,
-                          getOwnPropertyDescriptors(original));
+  const desc = getOwnPropertyDescriptors(original);
```

Comment:
I think maybe we should just be getting and setting the one descriptor, instead of all of them?

Reply:
I find that we are drifting off-topic here; changing all descriptors is the way the library is currently implemented. 
I am not changing this. IMH, modifying the way the callbackifed method property descriptors are set should be done in a separate PR.